pkg/tfbridge/x/tokens.go

```diff
@@ -472,20 +472,33 @@ type alias[T ~string] struct {
 }
 
 type aliasHistory struct {
-	Resources   map[string]*tokenHistory[tokens.Type]         `json:"resources"`
+	Resources   map[string]*resourceHistory                   `json:"resources"`
```

Comment:
So this is a change to a serializable struct that does not look bw-compat.. will it break parsing existing JSON data that was serialized using the previous form?

Reply:
Ah never mind, it might be bw-compat because tokenHistory[tokens.Type] is embedded.

Reply:
It is backwards compatible via embeddings, but thats good to call out.